### PR TITLE
expose flows

### PR DESCRIPTION
**Pull request by matehuszarik**

### CODE REVIEW

**Review by bmz1 — APPROVED** [review]

File: openapi/2023-12.yaml [hunk 1]
```diff
@@ -365,6 +428,35 @@ components:
...
+    Flow:
+      type:
+        - object
+      required:
+        - type
+        - attributes
+      properties:
+        type:
+          type: string
+          example: "flows"
```
Comment (bmz1) [inline openapi/2023-12.yaml:440]: Nit: not just here, but everywhere we use type, there is only on possible value. We could add an `enum: ['flows']` property.

File: openapi/2023-12.yaml [hunk 2]
```diff
@@ -365,6 +428,35 @@ components:
...
+              description: ID of flow
+              example: "655637c97825e6ae655fe1b5"
+            name:
+              type: string
+              description: Name of flow
+              example: "Welcome flow"
+            tags:
+              type: array
+              description: Flow tags
+              example: ["automated", "welcome"]
```
Comment (istvankovacs) [inline openapi/2023-12.yaml:459]: I'd add one more property to the list, whether the flow is active or not. That could be useful.

Other than that it looks good.

Comment (matehuszarik) [reply openapi/2023-12.yaml:459]: It's a bit more verbose than that. I was thinking about it but there are different statuses for automated flows and campaigns. Are we ok with exposing our [statuses](https://github.com/ghostmonitor/sequence/blob/master/src/api/models/sequence.model.ts#L37)?

Comment (istvankovacs) [reply openapi/2023-12.yaml:459]: Yes, that is what I thought about.

File: openapi/2023-12.yaml (hunk 1 above)
Comment (matehuszarik) [reply openapi/2023-12.yaml:440]: good idea

**Review by istvankovacs — APPROVED** [review]

**Pull request merged**